string sequence_type
string sequence_goal
---
bool success
---
int8 sequence_index
